spec/models/publication_spec.rb

```diff
@@ -346,19 +346,25 @@
   end
 
   describe 'update_manual_pub_from_pub_hash' do
+    let(:pub) { Publication.build_new_manual_publication({ a: :b }, 'some string') }
```

Comment:
:+1:  -  although ? about removing the last param that was there before the previous commit.